examples/intro_tutorial.ipynb

```diff
@@ -0,0 +1,588 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## HarmonyPy Introduction\n",
+    "\n",
+    "This notebook demonstrates several basic examples highlighting how to query and access customized data outputs from NASA Earthdata Harmony. See https://harmony-py.readthedocs.io/en/latest/ for detailed documentation on HarmonyPy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import packages\n",
+    "\n",
+    "First we import packages needed to request and visualize our data, as well as the `harmony-py` library itself. Make sure to install `harmony-py` and its dependencies into your current Python environment prior to executing the notebook:\n",
+    "\n",
+    "```  \n",
+    "$ pip install -U harmony-py \n",
+    "\n",
+    "```\n",
+    "\n",
+    "You can install this and any other necessary libraries directly into your Jupyter notebook Python kernel as well:\n",
+    "\n",
+    "```\n",
+    "!{sys.executable} -m pip install -U harmony-py\n",
+    "````\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !{sys.executable} -m pip install -U harmony-py #install harmony-py into Python kernel\n",
+    "import sys; sys.path.append('..')\n",
+    "import datetime as dt\n",
+    "from IPython.display import display, JSON\n",
+    "import rasterio\n",
+    "import rasterio.plot\n",
+    "\n",
+    "from harmony import BBox, Client, Collection, Request\n",
+    "from harmony.config import Environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Quick start\n",
+    "\n",
+    "You can request and download data using `harmony-py` in just a few easy lines. Although more advanced subsetting and transformation options may be supported on your data product of interest, this example below demonstrates a basic spatial bounding box and temporal range request, using the direct download method: \n",
+    "\n",
+    "```\n",
+    "harmony_client = Client(auth=('EDL_username', 'EDL_password'))\n",
+    "request = Request(\n",
+    "    collection=Collection(id=dataset_short_name),\n",
+    "    spatial=BBox(w, s, e, n),\n",
+    "    temporal={\n",
+    "        'start': dt.datetime(yyyy, mm, dd),\n",
+    "        'stop': dt.datetime(yyyy, mm, dd)\n",
+    "    }\n",
+    ")\n",
+    "job_id = harmony_client.submit(request)\n",
+    "results = harmony_client.download_all(job_id, directory='/tmp', overwrite=True)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The guidance below offers more detailed examples highlighting many of the helpful features provided by the Harmony Py library, including direct s3 access from Earthdata Cloud-hosted data if running in the AWS us-west-2 region."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```

Comment:
This section, which talks about authentication options, could also refer to entering credentials in a .env file. It operates in a similar fashion to .netrc as the credentials are stored in cleartext so I guess it becomes a matter of preference. The .env file is searched for in the current working directory, must be named `.env` and would have the format:
EDL_USERNAME=myusername
EDL_PASSWORD=mypass

Reply:
Thanks for this suggestion! I just pushed an update with that guidance on creating a .env file.